rust/main/Cargo.toml

```diff
@@ -129,6 +134,15 @@ solana-account-decoder = "=1.14.13"
 solana-client = "=1.14.13"
 solana-sdk = "=1.14.13"
 solana-transaction-status = "=1.14.13"
+solana-zk-token-sdk = "=1.14.13"
+spl-associated-token-account = { version = "=1.1.2", features = [
+  "no-entrypoint",
+] }
+spl-noop = { version = "=0.1.3", features = ["no-entrypoint"] }
+spl-token = { version = "=3.5.0", features = ["no-entrypoint"] }
+spl-token-2022 = { version = "=0.5.0", features = ["no-entrypoint"] }
```

Comment:
I wonder why we need this solana related dependencies here? It looks like starknet package does not use them.